### PR TITLE
Update CLI new app command and directory path

### PR DESCRIPTION
**Pull request by zachgiordano**
----
📚 Documentation preview 📚: https://griptape--189.org.readthedocs.build/en/189/

### CODE REVIEW

**Review by kyleroche — COMMENTED** [review]
works

**Review by kyleroche — COMMENTED** [review]
works

File: docs/griptape-cloud/getting-started/quickstart-cli.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ Next, we can use the Griptape CLI to scaffold a sample project that is compatibl
 You should see something like the following. In this example, I'm creating an app called _demo_app_ in my Home directory. I'm then verifying its contents via `ls`.
```
Comment (collindutter) [inline docs/griptape-cloud/getting-started/quickstart-cli.md:36]: > In this example, I'm creating an app called _demo_app_ in my Home directory

Might want to mention the `workplace` directory.

File: docs/griptape-cloud/getting-started/quickstart-cli.md [hunk 2]
```diff
@@ -36,13 +36,13 @@ Next, we can use the Griptape CLI to scaffold a sample project that is compatibl
...
 
 ```shell
-gt app new --directory ~/demo_app
+gt app new --directory ~/workplace demo_app
```
Comment (collindutter) [inline docs/griptape-cloud/getting-started/quickstart-cli.md:39]: Workplace or Workspace? Now I don't know which I normally use..

File: docs/griptape-cloud/getting-started/quickstart-cli.md (hunk 1 above)
Comment (zachgiordano) [reply docs/griptape-cloud/getting-started/quickstart-cli.md]: Good catch, updated

File: docs/griptape-cloud/getting-started/quickstart-cli.md (hunk 2 above)
Comment (zachgiordano) [reply docs/griptape-cloud/getting-started/quickstart-cli.md:39]: Just checked, looks like I'm on team `workplace`.

**Review by collindutter — APPROVED** [review]

File: docs/griptape-cloud/getting-started/quickstart-cli.md [hunk 3]
```diff
@@ -33,16 +33,16 @@ Step 3. **gt app new --directory ~/workplace demo_app**
 
 Next, we can use the Griptape CLI to scaffold a sample project that is compatible with Griptape Cloud and has a default structure in place. This is a quick way to get started if you aren't converting an existing Python project to Griptape.
 
-You should see something like the following. In this example, I'm creating an app called _demo_app_ in my Home directory. I'm then verifying its contents via `ls`.
+You should see something like the following. This example is creating an app called _demo_app_ in the workplace directory. The contents can be verified via `ls`.
```
Comment (vachillo) [inline docs/griptape-cloud/getting-started/quickstart-cli.md:36]: nit: maybe just call out here it creates a directory with that name as well.

**Review by vachillo — APPROVED** [review]

**Pull request merged**